src/jobs/test.yml

```diff
@@ -35,6 +35,18 @@ parameters:
       Optionally supply a custom package installation command, with any additional flags needed.
     type: string
     default: ""
+  test-results-path:
+    type: string
+    default: ""
+    description: |
+      If you use test-results-for, this will be configured automatically.
+      If you configure test results yourself (for example in your node config files) set this to the location that you write them to, and they will be uploaded.
+  test-results-for:
+    default: none
+    type: enum
+    enum: ["jest", "none"]
```

Comment:
Should we say "other" rather than none? I imagine we might want to cover more testing libraries in the future that have specific configuration.

Reply:
Good call. Replace `none` with `other`.